lib/parsers/txt.mjs

```diff
@@ -99,7 +99,8 @@ const licenseDeclarations = {
   trust_12_sep_2009_section6_b_p3: /This document is subject to BCP 78 and the IETF Trust's Legal Provisions Relating to IETF Documents \(https:\/\/trustee\.ietf\.org\/license-info\) in effect on the date of publication of this document\. Please review these documents carefully, as they describe your rights and restrictions with respect to this document\. Code Components extracted from this document must include Revised BSD License text as described in Section 4\.e of the Trust Legal Provisions and are provided without warranty as described in the BSD License\./,
   trust_28_dec_2009_section6_b_i_p3: /This document is subject to BCP 78 and the IETF Trust's Legal Provisions Relating to IETF Documents \(https:\/\/trustee\.ietf\.org\/license-info\) in effect on the date of publication of this document\. Please review these documents carefully, as they describe your rights and restrictions with respect to this document\. Code Components extracted from this document must include Revised BSD License text as described in Section 4\.e of the Trust Legal Provisions and are provided without warranty as described in the Revised BSD License\./,
   general_disclaimer: /This document and the information contained herein are provided on an "AS IS" basis and THE INTERNET SOCIETY AND THE INTERNET ENGINEERING TASK FORCE DISCLAIMS ALL WARRANTIES, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTY THAT THE USE OF THE INFORMATION HEREIN WILL NOT INFRINGE ANY RIGHTS OR ANY IMPLIED WARRANTIES OF MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE\./,
-  revised_bsd_license: /Code Components extracted from this document must include Revised BSD License text as described in Section 4\.e of the Trust Legal Provisions and are provided without warranty as described in the Revised BSD License\./
+  revised_bsd_license: /Code Components extracted from this document must include Revised BSD License text as described in Section 4\.e of the Trust Legal Provisions and are provided without warranty as described in the Revised BSD License\./,
+  draft_paragraph_pointing_to_the_list_of_current_ids: /The list of current Internet-\s*Drafts is at https:\/\/datatracker\.ietf\.org\/drafts\/current\/|The list of current Internet-Drafts can be accessed at http:\/\/www\.ietf\.org\/ietf\/1id-abstracts\.txt\./gi
```

Comment:
Again, why is there this `\s*\ in _one_ place. I suspect it's trying to account for newlines in a specific bit of input text, but that kind of whitespace should already be normalized before this check.